.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### What does this PR do?
+
+[Issue/Story](LINK_TO_STORY)
+
+### Why are we doing this? Any context or related work?
+
+### Where should a reviewer start
```

Comment:
Only manual testing steps did not need to be a question, this one does need to be